index.bs

```diff
@@ -303,6 +284,61 @@ Loader instances are initially created with the internal slots described in the
 
 <h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
 
+<h4 id="registry-GetCurrentStage" aoid="GetCurrentStage">GetCurrentStage(entry)</h4>
```

Comment:
a couple of abstract operations to interact with the pipeline, which is the ordered list of stages.